lib/action_policy/policy/authorization.rb

```diff
@@ -49,11 +49,15 @@ def initialize(*args, **params)
         @authorization_context = {}
 
         self.class.authorization_targets.each do |id, opts|
-          raise AuthorizationContextMissing, id unless params.key?(id)
+          if opts[:optional] == true
+            val = nil
```

Comment:
Wouldn't it clear the value if it's passed? Can we add a test with `OptionalRoleTestPolicy.new(role: some_role)`?

Reply:
True. My bad.
Already fixed 👌🏻